### PR TITLE
Sets: Enabled tests, improved documentation, minor cleanup

### PR DESCRIPTION
**Pull request by oscargus**
#### Brief description of what is fixed or changed
Found two disabled tests that could be enabled, but since that seems a bit small for a PR I also improved the formatting of the documentation (not the actual content, though, as I did it during a meeting, so no time to think, just edit...) and cleaned the code a bit.

#### Other comments
I do not know how to build the documentation (the Makefile doesn't work on my computer), so not really sure if the formatting is always correct.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/sets/ordinals.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
 from sympy.core import Basic, Integer
 import operator
 
+
 class OmegaPower(Basic):
     """
     Represents ordinal exponential and multiplication terms one of the
-    building blocks of the Ordinal class.
-    In OmegaPower(a, b) a represents exponent and b represents multiplicity.
+    building blocks of the :class:`Oridnal` class.
```
Comment (oscarbenjamin) [inline sympy/sets/ordinals.py:8]: Ordinal

File: sympy/sets/fancysets.py [hunk 2]
```diff
@@ -1002,12 +1002,12 @@ def as_relational(self, x):
...
-    """
-    Normalize a Real Set `theta` in the Interval [0, 2*pi). It returns
+    r"""
+    Normalize a Real Set `theta` in the interval `[0, 2\pi)`. It returns
     a normalized value of theta in the Set. For Interval, a maximum of
-    one cycle [0, 2*pi], is returned i.e. for theta equal to [0, 10*pi],
-    returned normalized value would be [0, 2*pi). As of now intervals
-    with end points as non-multiples of `pi` is not supported.
+    one cycle `[0, 2\pi]`, is returned i.e. for theta equal to `[0, 10\pi]`,
+    returned normalized value would be `[0, 2\pi)`. As of now intervals
```
Comment (oscarbenjamin) [inline sympy/sets/fancysets.py:1009]: I guess that for latex `$x$` is preferred now rather than single backticks

**Comment by oscarbenjamin** [comment]
Looks good to me

File: sympy/sets/ordinals.py (hunk 1 above)
Comment (smichr) [inline sympy/sets/ordinals.py:8]:
```suggestion
    building blocks of the :class:`Ordinal` class.
```

File: sympy/sets/fancysets.py (hunk 2 above)
Comment (smichr) [inline sympy/sets/fancysets.py:1009]:
```suggestion
    one cycle $[0, 2\pi]$, is returned i.e. for theta equal to $[0, 10\pi]$,
    returned normalized value would be $[0, 2\pi)$. As of now intervals
```

**Pull request merged**